### PR TITLE
Cluster autoscaler doc for 1.3

### PR DESCRIPTION
**Pull request by mwielgus**
Same as #677 but for release-1.3 branch

cc: @piosz @fgrzadkowski @jszczepkowski

### CODE REVIEW

File: docs/admin/cluster-management.md [hunk 1]
```diff
@@ -66,37 +66,52 @@ Instance Group will take care of putting appropriate image on new machines and s
...
-### Horizontal auto-scaling of nodes (GCE)
+### Cluster autoscaling
 
-If you are using GCE, you can configure your cluster so that the number of nodes will be automatically scaled based on:
+If you are using GCE or GKE, you can configure your cluster so that it is automatically rescaled based on
+pod needs.
 
- * CPU and memory utilization.
- * Amount of of CPU and memory requested by the pods (called also reservation).
+As desribed in [Compute Resource](../user-guide/compute-resources.md), user can declare how much
```
Comment (johndmulhausen) [inline docs/admin/cluster-management.md]: Proper link url: (/docs/user-guide/compute-resources/)

File: docs/admin/cluster-management.md [hunk 2]
```diff
@@ -66,37 +66,52 @@ Instance Group will take care of putting appropriate image on new machines and s
...
+CPU and memory a pod needs to run. This amount is called request and is used by Kubernetes scheduler (along with ports
+and labels, if provided) to find a place to start the pod. If there is no node that has enough free capacity or
+doesn't match other pod requirements then the pod has to wait, possibly quite long,
+until some pods are terminated. Or until a new node is added.
```
Comment (johndmulhausen) [inline docs/admin/cluster-management.md]: How about:

As described in [Compute Resource](/docs/user-guide/compute-resources/), users can reserve how much CPU and memory is allocated to pods. This information is used by the Kubernetes scheduler to find a place to run the pod. If there is no node that has enough free capacity (or doesn't match other pod requirements) then the pod has to wait until some pods are terminated or a new node is added.

Comment (mwielgus) [inline docs/admin/cluster-management.md]: Done.

File: docs/admin/cluster-management.md (hunk 1 above)
Comment (mwielgus) [inline docs/admin/cluster-management.md]: Done.

**Comment by johndmulhausen** [comment]
I'm closing this PR as it doesn't have the review history. Thank you  - this was the correct branch to work in for 1.3 work but I don't think it's a good idea to duplicate PRs like this.

**Comment by johndmulhausen** [comment]
Tech Review LGTM was given in the other PR and this looks good to me here.

**Pull request merged**